lib/engine/game/g_1848/game.rb

```diff
@@ -596,6 +604,7 @@ def operating_round(round_num)
             G1848::Step::SpecialBuyTrain,
             G1848::Step::BuyTrain,
             [G1848::Step::BuyCompany, { blocks: true }],
+            [G1848::Step::Loan, { blocks: true }],
```

Comment:
Do you want to make the person pass twice in a row? There are better ways to only block once at the end of the turn. Also, will the loan step block if the company has already taken its loan?

Reply:
one pass is better than two :-). How do you only block once?

Reply:
you're right, it doesn't. It allows a company to take two loans in one OR, which isn't not allowed. Oops. Good catch

Reply:
Common practice is to tie both capabilities into one step (See payoff loan in 1817). Maybe EndOfTurnActions?